content/200-concepts/100-components/02-prisma-client/053-client-extensions/index.mdx

```diff
@@ -196,3 +196,7 @@ if (xPrisma.$connect) {
   xPrisma.$connect()
 }
 ```
+
+### Usage with nested operations
+
+Prisma Client extensions currently do not support nested read and write operations. If you're interested in this feature, let us know by opening a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) with your use case. 
```

Comment:
We don't really want separate feature requests for this.
We should have one already from @SevInf probably, and ask for comment with use case on that.